### PR TITLE
Removed unsupported versions from PEM and EDB*Plus

### PR DESCRIPTION
**Pull request by dwicinas**
Removed generated files from EDB*Plus and PEM 8 and 9

### CODE REVIEW

File: product_docs/docs/pem/8/installing/dependencies.mdx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+Typically, PEM is built against the latest version of each dependency available from the vendor repository for a given platform and architecture.
+In some cases, PEM requires a newer version of a library than is available in the vendor repository.
+In these cases a newer version of the package, prefixed with `edb-` is sourced from EDB's repositories.
+
+!!! Note
+This information is provided for reference. Packages from vendor repositories are not supported or patched by EDB. 
+Refer to your operating system documentation or support provider for details of these packages.  
+
+Because these dependencies are updated frequently, the tables below are valid only for the latest patch release of PEM.
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing/dependencies.mdx:23]: I don't think this page should exist in PEM 8. In any case, PEM 8 is EOL so we will be looking to remove the docs very soon. We could do that now if it helps with this change?

File: product_docs/docs/pem/8/installing/linux_x86_64/index.mdx [hunk 2]
```diff
@@ -13,16 +13,12 @@ redirects:
 navigation:
   - pem_rhel_9
   - pem_rhel_8
-  - pem_rhel_7
   - pem_other_linux_9
   - pem_other_linux_8
-  - pem_centos_7
   - pem_sles_15
-  - pem_sles_12
```
Comment (sonotley) [inline product_docs/docs/pem/8/installing/linux_x86_64/index.mdx:21]: SLES is still in support until November isn't it?

**Review by sonotley — CHANGES_REQUESTED** [review]

File: product_docs/docs/pem/8/installing/linux_x86_64/index.mdx (hunk 2 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/installing/linux_x86_64/index.mdx:21]: Yes, it is. We will reinstate SLES 12 changes back in sometime

File: product_docs/docs/pem/8/installing/dependencies.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/installing/dependencies.mdx]: deleted

**Review by sonotley — APPROVED** [review]

**Pull request merged**